src/sage/graphs/graph_decompositions/tree_decomposition.pyx

```diff
@@ -852,6 +852,22 @@ def make_nice_tree_decomposition(graph, tree_decomp):
         sage: bip_one_four_TD = bip_one_four.treewidth(certificate=True)
         sage: make_nice_tree_decomposition(bip_one_four, bip_one_four_TD)
         Nice tree decomposition of Tree decomposition: Graph on 15 vertices
+
+    The following two examples are for testing GitHub Issue 36843::
+
+        sage: from sage.graphs.graph_decompositions.tree_decomposition import make_nice_tree_decomposition
+        sage: triangle = graphs.CompleteGraph(3)
+        sage: triangle_TD = triangle.treewidth(certificate=True)
+        sage: make_nice_tree_decomposition(triangle, triangle_TD)
+        Nice tree decomposition of Tree decomposition: Graph on 7 vertices
+
+    ::
+
+        sage: from sage.graphs.graph_decompositions.tree_decomposition import make_nice_tree_decomposition
+        sage: graph = graphs.CompleteBipartiteGraph(2, 5)
+        sage: graph_TD = graph.treewidth(certificate=True)
+        sage: make_nice_tree_decomposition(graph, graph_TD)
+        Nice tree decomposition of Tree decomposition: Graph on 25 vertices
```

Comment:
Could you also add a `TESTS` block to check simple cases, like empty graph, graph with a single vertex, with 2 vertices connected by an edge.

Reply:
> Could you also add a `TESTS` block to check simple cases, like empty graph, graph with a single vertex, with 2 vertices connected by an edge.

I'm slightly confused by "graph with a single vertex, with 2 vertices connected by an edge".

Do you mean a singleton and a K2?

Or do you mean a triangle with two edges removed?

Reply:
Yes, a singleton and a K2.